browser/extensions/brave_extension_management.cc

```diff
@@ -85,7 +86,17 @@ void BraveExtensionManagement::OnTorDisabledChanged() {
 #if BUILDFLAG(ENABLE_TOR)
   if (TorProfileServiceFactory::IsTorDisabled()) {
     TorProfileManager::GetInstance().CloseAllTorWindows();
-    g_brave_browser_process->tor_client_updater()->Cleanup();
+    if (g_brave_browser_process->tor_client_updater()) {
+      g_brave_browser_process->tor_client_updater()->Cleanup();
+    }
+    if (g_brave_browser_process->tor_pluggable_transport_updater()) {
+      g_brave_browser_process->tor_pluggable_transport_updater()->Cleanup();
+    }
+  } else if (TorProfileServiceFactory::GetTorBridgesConfig().use_bridges ==
+             tor::BridgesConfig::Usage::kNotUsed) {
+    if (g_brave_browser_process->tor_pluggable_transport_updater()) {
+      g_brave_browser_process->tor_pluggable_transport_updater()->Cleanup();
+    }
```

Comment:
This seems wrong, you should use `PrefChangeRegistrar` to listen for changes of `tor::prefs::kBridgesConfig` separately in different callback instead of checking it in `OnTorDisabledChanged`

Reply:
You're right. fixed.